lang/python/python-cryptography/Makefile

```diff
@@ -32,18 +32,18 @@ define Package/python3-cryptography
   URL:=https://github.com/pyca/cryptography
   DEPENDS:= \
       +libopenssl \
+      +libopenssl-legacy \
```

Comment:
Is this really necessary?  I'd prefer to make due without... they've been marked for deprecation for a reason.

Reply:
cryptography package deliberately continues to support legacy algorithms and requires an [environment variable](https://cryptography.io/en/latest/openssl/#legacy-provider-in-openssl-3-x) to be set to bypass this.

I don't think it would be a good idea to patch this out because other programs/packages that use cryptography will also expect legacy algorithms to be available.

Reply:
I guess it will get retired upstream in due time.